### PR TITLE
fix(agent): Support base profile on GB300

### PR DESCRIPTION
**Pull request by liamy-nv**
## Summary
- Keep H100 Base on Nano v2
- On GB300, generate Nemotron 3.5 Lightning plus the existing SBSA RT-VLM tag so Base can start on ARM64.

## Checklist
- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/blob/HEAD/CONTRIBUTING.md).
- [ ] I have installed and run [pre-commit hooks](https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/blob/HEAD/CONTRIBUTING.md#1-enable-pre-commit-hooks) locally (`uv run pre-commit install` once, then hooks run on every `git commit`).
- [ ] Every commit on this PR is [DCO sign-off](https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/blob/HEAD/CONTRIBUTING.md#developer-certificate-of-origin-dco)'d (`git commit -s` adds a `Signed-off-by` trailer that certifies you have the right to submit the change under Apache-2.0).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.
- [ ] No secrets, API keys, or credentials committed.

### CODE REVIEW

File: deploy/docker/developer-profiles/dev-profile-base/vss-agent/configs/config.yml [hunk 1]
```diff
@@ -248,6 +248,19 @@ workflow:
...
   - report_agent
   # yamllint disable rule:line-length
+  # Keep named report requests deterministic for models that over-apply the
+  # top agent's default clarification examples.
+  plan_prompt: |
+    Review the available tools, the conversation history, and the user's question, then produce a concise numbered execution plan. Start each step with a tool name and a brief description of the step.
+    Put relevant context (for example, sensor IDs or time ranges) from the conversation history directly in the plan steps so the execution agent does not need to re-read the history.
+    A non-empty name supplied in the request is the sensor_id. For example, in "Generate a report for warehouse_sample", the sensor_id is "warehouse_sample"; this request is complete and MUST NOT trigger a clarifying question.
+    If the user requests a report and supplies a name, the plan MUST contain exactly one action: call `report_agent` directly with that exact name as sensor_id and the original request as user_query. Do not verify the name or media type first.
+    Ask which video or camera the user means only when the request contains no name and the conversation provides no usable name.
```
[the remark this answers is not in the document]
Comment (liamy-nv) [reply deploy/docker/developer-profiles/dev-profile-base/vss-agent/configs/config.yml:258]: this is an edge case that is not a concern for base profile. stream-type routing is an LVS concern and is already documented on the LVS execution prompt

**Comment by liamy-nv** [comment]
/ok to test [95336a6](https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/pull/1825/commits/95336a6b82f427a90e7992c0a8de1b6886b3356a)

**Review by zac-wang-nv — COMMENTED** [review]
_This is Hinton, the VSS CI OSRB reviewer_

## Automated OSRB Review — INCONCLUSIVE

The internal review failed closed: required evidence collection had errors

- Dependency changes evaluated: `0`
- Public evidence: [License Diff Actions run](https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/actions/runs/32800488449)
- Internal approval evidence was evaluated by the protected compliance pipeline.

File: deploy/docker/developer-profiles/dev-profile-base/overrides.env [hunk 2]
```diff
@@ -50,6 +50,7 @@ VLM_MODE=local_shared
 # - nvidia/llama-3.3-nemotron-super-49b-v1.5 -> llama-3.3-nemotron-super-49b-v1.5
 # - openai/gpt-oss-20b -> gpt-oss-20b
 # Select the local LLM model name, or the model id exposed by a remote /v1/models endpoint.
+# GB300 selects Nemotron 3.5 Lightning in generated.env; other hardware keeps this Nano v2 default.
 LLM_NAME=nvidia/nvidia-nemotron-nano-9b-v2
```
Comment (zac-wang-nv) [inline deploy/docker/developer-profiles/dev-profile-base/overrides.env]: can you change the default to 3.5 lightening? 
https://github.com/NVIDIA-AI-Blueprints/video-search-and-summarization/blob/develop/deploy/docker/developer-profiles/dev-profile-lvs/overrides.env

**Review by zac-wang-nv — APPROVED** [review]
